### ISSUE TITLE
Project Status?

### ISSUE THREAD

**Issue by cameron-emburse** [danielquinn/django-encrypted-filefield#22]
Is this project still maintained?

**Comment by AndrewGrossman** [comment]
+1

**Comment by nanorepublica** [comment]
Yes, but not actively worked on since it is stable and does the job it needs to.

**Comment by cameron-emburse** [comment]
Does this support newer versions of Django (e.g., at least 4.2)?

**Comment by nanorepublica** [comment]
It doesn't explicitly as I have yet to update the main codebase with the project to 4.2 (need to do database upgrades first). That said I have no reason to think it would break for newer versions of Django. 

I am also very happy to take PRs if there are changes that are required.
